build.sbt

```diff
@@ -112,6 +112,23 @@ inThisBuild(
     resolvers += Resolver.sonatypeRepo("public"),
     resolvers += Resolver.sonatypeRepo("snapshot"),
     dependencyOverrides += V.guava,
+    dependencyOverrides += "org.eclipse.platform" % "org.eclipse.ant.core" % "3.5.500",
```

Comment:
Uff, this is a lot, thanks for compiling the list!

Reply:
I've had more fun afternoons

Reply:
If possible can we add a note here as to why we are doing this? I can see later on someone jumping in here and looking at this and wondering what in the world we are doing here with all these overrides.

Reply:
I was thinking of moving it later to a separate file and commenting. This PR is pretty huge and I don't want to add anything more right now.

Reply:
Also, I think you did another comment and I can't find it :sweat_smile:  which is a good indicator that this got way to big. What do you think? Is it ok to do same follow ups later?

Reply:
:) of course, a follow-up is totally fine.

For the other comment 😆 I deleted it. A couple weeks ago I started reviewing this and only got like half way through and didn't realize I had draft comments saved, so when I added this comment it also posted those. They aren't relevant anymore, so I just removed them. Sorry for the noise!